didx-xyz/yoma-mobile
# DRAFT: Yoma-590 about add view state

## Scope [YOMA-590](https://yomaworld.atlassian.net/browse/YOMA-590)

![](https://media.giphy.com/media/xUPGcj4nkFIZ3zGFLW/giphy.gif)

## Work Done
- a ton of rewiring of the widgets, cleaning up the navigation types, some other standardising of links.
- fixing up the add/edit buttons.
- creating a view section for about details that's truncated in the widget
- adding a view section for about
- fixed up the form section in about to save the biography.

## Steps to Test
- navigate into About widget, click the edit - and should see edit screen. Click the about or 'view all' and it should load the view screen.
